PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for being here!
+Review these resources before making a contribution:
+  - Code of Conduct: https://github.com/target/markdown-inject/blob/main/CODE_OF_CONDUCT.md
+  - Contribution guide: https://github.com/target/markdown-inject/blob/main/CONTRIBUTING.md
+-->
+
+closes #<issue number>
+
+# Changes
+
+<!-- What did you change? -->
+
+<!-- Why did you change it? What value does it bring? -->
+
+# Validation
+
+<!--
+How do we know these changes do what they are intended to?
+Explain things that you did and we can do.
+Examples:
+  - Automated tests that validate the change
+  - Documentation that explains the change
+  - Manual steps that can be performed to validate the change
+    - Manual steps are icing on the cake. Do not rely on manual validation to do what automation or documentation can do better.
```

Comment:
This rolls over into the next line in the github editor and I'm not in love with that, but I'll let it go because I don't want to break the single thought into 2 bullets.

Reply:
would it be more appropriate to say `last resort` vs `icing on the cake?` 

neither really captures what the second sentence conveys IMO

Reply:
I'm not really sure how to say "we love these but don't do just these"